game-state-service

```diff
@@ -112,10 +126,18 @@ class GameStateService(Gio.Application):
           <arg type='s' name='key' direction='in'/>
           <arg type='v' name='value' direction='in'/>
         </method>
+        <method name='Delete'>
+          <arg type='s' name='key' direction='in'/>
+        </method>
+        <method name='DeleteAll'>
+        </method>
         <signal name='changed'>
           <arg type='s' name='key'/>
           <arg type='v' name='value'/>
         </signal>
+        <signal name='deleted'>
+          <arg type='s' name='key'/>
```

Comment:
If you make this an array, you won't have to emit a gazillion signals when `DeleteAll()` is called.